### ISSUE TITLE
Username or password is incorrect. (Command: *sensitive*; response: #2# [BAD, Missing""])

### ISSUE THREAD

**Issue by LanceHaverkamp** [k9mail/k-9#4414]
Using K9 on Android phone & Chromebook
Same accounts, same settings, current versions from play.google.com.  Works on phone, but getting:
Username or password is incorrect. (Command: *sensitive*; response: #2# [BAD, Missing""])
error on Chromebook, as soon as I copy/paste password.

Is this a known issue, or do I need to try to find version info on this chromebook?
Thanks,

**Comment by cketti** [comment]
Try entering the password manually. It's possible that by pasting the password you end up with additional characters that shouldn't go into the password input field.

If that doesn't work please create a new issue and include a debug log with sensitive logging enabled. This will contain your password. So you should change it after recording the debug log.

**Comment by eickeh** [comment]
Hi there, just a note for everyone experiencing the same issue and also going crazy. 
I just ran into the same issue, also same error message. Entering the password manually did indeed work for me. 

However, the password in the clipboard wasn't wrong itself. It just seems that Android screws up the password somehow when pasting in password fields. I copied the password and pasted it in several different apps (**without** clearing the clipboard in between, so it indeed was the same content). Here's what I got:
- All browser-based password forms worked as expected, i.e. several websites on Firefox and Login in Nextcloud App 
- But pasting it into K-9 and Schildichat just didn't work.
